Add examples how to stream files

D-Marc1 opened the pull request:
Show examples for both single and multiple files, along with an explanation for how to use `FormData` text before processing the file.

D-Marc1 committed:
Add examples how to stream files

Show examples for both single and multiple files, along with an explanation for how to use `FormData` text before processing the file.

thetutlage commented on docs/docs/guides/01-basics/03-http/07-file-uploads.md:
````diff
@@ -125,6 +125,81 @@ await avatar.move(Application.tmpPath('uploads'), {
+
+```ts
+request.multipart.onFile('file', {}, async (file) => {
+  AWS.config.update({
````
Shouldn't this be happening once at the application boot level vs doing it on every request?

thetutlage commented on docs/docs/guides/01-basics/03-http/07-file-uploads.md:
```diff
@@ -125,6 +125,81 @@ await avatar.move(Application.tmpPath('uploads'), {
+
+const uploadPromises = [];
+
+files.forEach((file, index) => {
```
Using `map` is a better option here

D-Marc1 committed:
Update 07-file-uploads.md

Ok, I think I resolved your request changes. Let me know if I missed something.

I also added a section for multiple files with distinct file names as well.

D-Marc1 committed:
Fix Multiple Files with Distinct File Names

Show how to separate where the upload would upload to,

D-Marc1 committed:
Minor grammatical fix

Just added a comma.

D-Marc1 committed:
Check if promises in promise.all() has length > 0

D-Marc1 committed:
Add optional chaining in case no file uploaded

- Add optional chaining in case no file uploaded
- Add request.multipart?.process() to single file uplaod
